extensions/amp-story-shopping/0.1/amp-story-shopping-tag.css

```diff
@@ -60,7 +60,7 @@ amp-story-shopping-tag {
   justify-content: center !important;
   background-color: rgba(0, 0, 0, 0.6) !important;
   border-radius: 100% !important;
-  background-image: url('data:image/svg+xml;charset=utf-8,<svg width="12" height="12" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M11.49 5.755 6.24.505a1.16 1.16 0 0 0-.823-.338H1.334A1.17 1.17 0 0 0 .167 1.333v4.084c0 .32.128.612.344.828l5.25 5.25c.21.21.502.338.823.338.32 0 .612-.128.822-.344l4.083-4.083a1.14 1.14 0 0 0 .345-.823c0-.32-.134-.618-.345-.828ZM2.208 3.083a.874.874 0 1 1 0-1.75.874.874 0 1 1 0 1.75Z" fill="#fff"/></svg>') !important;
+  background-image: url('data:image/svg+xml;charset=utf-8,<svg width="12" height="12" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M11.49 5.755 6.24.505a1.16 1.16 0 0 0-.823-.338H1.334A1.17 1.17 0 0 0 .167 1.333v4.084c0 .32.128.612.344.828l5.25 5.25c.21.21.502.338.823.338.32 0 .612-.128.822-.344l4.083-4.083a1.14 1.14 0 0 0 .345-.823c0-.32-.134-.618-.345-.828ZM2.208 3.083a.874.874 0 1 1 0-1.75.874.874 0 1 1 0 1.75Z" fill="#fff"/></svg>');
```

Comment:
Was it necessary to remove the `!important` or would it be possible to keep it and also add an `!important` to the `amp-story-shopping-tag-pill-image` background image style in amp-story-shopping-tag.js? 

My impression is that keeping the `!important` would confine the background image styling of `amp-story-shopping-tag-pill-image` to the specified product-icon in the JSON, whereas removing it would allow for publishers to also specify the background image in their CSS. I'm not sure if my thoughts here are correct, just wondering what you think

Reply:
Good call. We can set `!important` inline :)